be/src/service/internal_service.cpp

```diff
@@ -135,38 +135,52 @@ void PInternalServiceImplBase<T>::_transmit_chunk(google::protobuf::RpcControlle
                                                   const PTransmitChunkParams* request, PTransmitChunkResult* response,
                                                   google::protobuf::Closure* done) {
     auto begin_ts = MonotonicNanos();
-    VLOG_ROW << "transmit data: " << (uint64_t)(request) << " fragment_instance_id=" << print_id(request->finst_id())
-             << " node=" << request->node_id() << " begin";
+    auto msg = "transmit data: " + std::to_string((uint64_t)(request)) +
+               " fragment_instance_id=" + print_id(request->finst_id()) +
+               " node = " + std::to_string(request->node_id());
```

Comment:
" node = " -> " node="

Reply:
concat log message to VLOG to avoid construct string when we disable VLOG

Reply:
fixed